src/main/java/com/WWI16AMA/backend_api/PlaneLog/PlaneLogEntry.java

```diff
@@ -0,0 +1,96 @@
+package com.WWI16AMA.backend_api.PlaneLog;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Pattern;
+import java.time.LocalDateTime;
+
+@Entity
+public class PlaneLogEntry {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private long id;
+    private LocalDateTime refuelDateTime;
+    @NotNull
+    private Integer memberId;
+    @NotNull
+    @Pattern(regexp = ".{0,35}")
```

Comment:
am besten nochmal prüfen, wie das Frontend das implementiert, die übertreiben gerne bissel. Vllt mal in der Frontend-Codebase nach "pattern" suchen. Die prüfen teilweise viel mehr als wir..